sandbox/src/services/mockResponseProvider.js

```diff
@@ -510,12 +510,12 @@ module.exports = {
       const id = request.params.id;
       const version = request.params.version
 
-      // Scenario 1 - ooked to directly-bookable service
+      // Scenario 1 - Booked to directly-bookable service
       if (id === '70000' && (version === undefined || version === '5')) {
```

Comment:
Very minor but the version ID provided should match the `versionId` included as part of the `meta` object within the `Appointment` returned. 70000 for example includes a `versionId` of `0` for example, so either this logic should be updated to match the examples or the examples should be updated to match the logic included here.

Reply:
Changed